CHANGELOG.md

```diff
@@ -43,6 +43,8 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 - The active node is highlighted with a "halo ring". Additionally, the node is also rendered as a circle. In flight and oblique modes the halo is hidden. [#3731](https://github.com/scalableminds/webknossos/pull/3731)
 - In the dashboard list of active tasks, the project name is now featured more prominently, as it switched places with the task type summary. [#3792](https://github.com/scalableminds/webknossos/pull/3792)
 - Isosurfaces are now loaded from the middle outwards. [#3818](https://github.com/scalableminds/webknossos/pull/3818)
+- The brush size will now be remembered across page reloads. [#3827](https://github.com/scalableminds/webknossos/pull/3827)
+- Do not show publication view if no publications are specified. [#3778](https://github.com/scalableminds/webknossos/pull/3778)
```

Comment:
Are changelog entries moved or copied to the highlights section? I think the first one is moved and the second one is copied? Or am I missing something here?

Reply:
The first one is moved to the changed-section, the second I just added.